### PR TITLE
Add exclude paths for Tests directories

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket #104425778](https://www.pivotaltracker.com/story/show/104425778): Tests and code can be found at various level in the src tree, all tools should be properly configured to run all tests; include all sourcecode files but exclude test files from static analysis.

### CODE REVIEW

**Comment by arothuis** [comment]
Review; @DRvanR, @rjkip

File: build.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
         <pathconvert property="changeset.php.raw" pathsep="${line.separator}">
             <fileset dir="src">
                 <include name="**/*.php"/>
+                <exclude name="**/Tests/**"/>
```
Comment (DRvanR) [inline build.xml]: Goes for most of these, what about `src/{VerificationSuite}/Tests/{SpecificTest}?

Comment (DRvanR) [inline build.xml]: Might be easier to just use `/Test/` ourselves then... thoughts?

Comment (arothuis) [inline build.xml]: Correct me if I'm wrong, but is that going to be an issue given the format we planned on using: `src/VerificationSuite/Metadata/Test/AvailabilityTest.php`?

Comment (DRvanR) [inline build.xml]: Guess not eh :wink:
--
ibuildings
WEB & MOBILE APP DEVELOPMENT

Daan van Renterghem
Software Engineer

phone (direct): +31 30 75 51 590
skype: dvrenterghem-ibuildings
e-mail: dvrenterghem@ibuildings.nl
phone: +31 88 00 24 000
www.ibuildings.nl

To all agreements with Ibuildings.nl BV, our general terms and conditions
<http://www.ibuildings.nl/algemene_voorwaarden> apply.

**Comment by DRvanR** [comment]
Did you verify that it works as intended for everything? I.e. created a file with violations for cs/md/etc. and ensured that they ignored it properly?

**Comment by arothuis** [comment]
Basic check: I created a file that had violations and I made it fail in included directories. I then moved it to a Tests dir and it passed (except phpunit was mad because of syntax errors).

**Comment by DRvanR** [comment]
Good enough for me, :+1:

**Pull request merged**